db/src/Pos/DB/Block/Epoch.hs

```diff
@@ -351,7 +352,12 @@ consolidateOneEpoch ccp epochSlots = do
 
     -- After the check point is written, delete old blunds for the epoch we have just
     -- consolidated.
-    lift $ mapM_ deleteOldBlund sihs
+    lift $ mapM_ deleter $ chunksOf 1000 sihs
+  where
+    deleter :: ConsolidateM ctx m => [SlotIndexHash] -> m ()
+    deleter xs = do
+        mapM_ deleteOldBlund xs
+        sleepSeconds 2
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/src/Pos/DB/Block/Epoch.hs#L355-L360